### PR TITLE
GRPC Ubuntu build script

### PR DESCRIPTION
**Pull request by lysannef**
@seth-priya Please find attached the grpc ubuntu 18.04 build script. 
Kindly let me know if any issues.

### CODE REVIEW

File: g/grpc/grpc_ubuntu_18.04.sh [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : GRPC
+# Version       : '1.33.2'
+# Source repo   : https://github.com/grpc/grpc
+# Tested on     : Ubuntu 18.04
+# Script License: MIT License
+# Maintainer    : Lysanne Fernandes <@us.ibm.com>
```
Comment (seth-priya) [inline g/grpc/grpc_ubuntu_18.04.sh:8]: @lysannef please correct the email address

Comment (lysannef) [reply g/grpc/grpc_ubuntu_18.04.sh]: @seth-priya  corrected email

**Review by seth-priya — COMMENTED** [review]
why are we skipping the test suites @lysannef ?

**Comment by seth-priya** [comment]
@lysannef as discussed please add comments explaining why we are skipping the tests and also squash the commits, thanks

**Comment by seth-priya** [comment]
@lysannef any update on this?

**Comment by lysannef** [comment]
@seth-priya  Done, added comment about skipping test cases.
Very sorry for the delay.

**Review by seth-priya — APPROVED** [review]

**Pull request merged**